solvers/+drakeFunction/+geometry/+test/testCartesian2CylindricalTransform.m

```diff
@@ -25,6 +25,7 @@
 cylinder_origin = randn(3,1);
 fcn3 = Cartesian2CylindricalTransform(cylinder_axis,cylinder_x_dir,cylinder_origin);
 x_cartesian = randn(6,1);
+x_cartesian(4:6) = rotmat2rpy(rpy2rotmat(x_cartesian(4:6)));
```

Comment:
might it be cleaner to use:
http://drake002.csail.mit.edu/drake/doxygen/uniformly_random_nonsingular_r_p_y_8m.html